src/FileReader/PHPFileReader.php

```diff
@@ -11,6 +12,8 @@
 
     public function read($filename)
     {
+        Assertion::string($filename);
+
         $this->filename = $filename;
 
         $this->assertFileExists();
```

Comment:
@localheinz have asserted for `file` on `$filename` but not keen on the idea of `isArray` on `$config` as it throws an `InvalidArgumentException` and I'm not convince that's right for the file contents. Thoughts?